notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -142,8 +145,7 @@
     "# It's possible to train FP32 model from scratch, but it might be slow. So the pre-trained weights are downloaded by default.\n",
     "pretrained_on_tiny_imagenet = True\n",
     "fp32_pth_url = \"https://storage.openvinotoolkit.org/repositories/nncf/openvino_notebook_ckpts/302_resnet18_fp32.pth\"\n",
-    "if pretrained_on_tiny_imagenet:\n",
-    "    urlretrieve(fp32_pth_url, fp32_pth_path)"
+    "download_file(fp32_pth_url, directory=MODEL_DIR, filename=fp32_checkpoint_filename)"
```

Comment:
It seems that instead of creating a new fp32_checkpoint_filename you could use `fp32_pth_path.name`  as filename

Reply:
@kshpv Please let me know if you have comments or are waiting for something. Using download_file is a nice improvement and I would like to merge this.

Reply:
Sorry for the late reply. I was busy with the other issues. I will fix it soon

Reply:
Thanks @kshpv no worries, I just did not want this PR to be forgotten. It is not urgent.